api/types/types.go

```diff
@@ -172,6 +172,14 @@ var ControllerPolicyRule = []rbacv1.PolicyRule{
 	},
 }
 
+var ControllerRouteIngressPolicyRule = []rbacv1.PolicyRule{
```

Comment:
The name here doesn't convey that it is a specialised aspect of route that is enabled, namely the explicit setting of the host.